### DISCUSSION TITLE
ERR_CERT_COMMON_NAME_INVALID when accessing the link to the project

### DISCUSSION THREAD

**Discussion by denvudd** [vercel/community#4237] [Help] [answered]
### Question

When I simply copy the link to my portfolio project (https://dmytro-yurin-portfolio.vercel.app/), the link opens. However, when I paste it into a PDF file (resume) or use it in Google Meet, for example, it shows an error:

```
NET::ERR_CERT_COMMON_NAME_INVALID
Subject: *.vercel.app

Issuer: R3

Expires on: Dec 24, 2023

Current date: Sep 25, 2023
```

How can it be fixed? I want users to be able to access the project from anywhere. The issue is not unique to me, as my friends also encounter the same problem when trying to open it, and the behavior seems similar.

[Resume file](https://cv.djinni.co/2e/3068448372d6afd5267e33218c5fb8/my_resume.pdf)

![image](https://github.com/vercel/community/assets/68691654/b2ca86ac-ba81-4d0f-bb1d-b9fb6f6cd3a3)

**Comment by swarnava** [comment]
Hi,

Can you send the screenshot of the error?

**Reply by denvudd** [reply]
Hi,

Sure. Here is the translation if you need it:
`Attackers may try to steal your data from the website www.dmytro-yurin-portfolio.vercel.app (for example, passwords, messages or bank card numbers).`


![image](https://github.com/vercel/community/assets/68691654/58b5238f-28dd-413a-b2be-5d5def20aff4)

**Reply by swarnava** [reply, marked as answer]
Hi there, can you try accessing without www? https://dmytro-yurin-portfolio.vercel.app/

www.dmytro-yurin-portfolio.vercel.app is a different domain
